utils/utils.go

```diff
@@ -208,6 +208,11 @@ func PopulateEndpointNets(wep *api.WorkloadEndpoint, result *current.Result) err
 	return nil
 }
 
+type CNITestArgs struct {
```

Comment:
I think this belongs in types.go alongside the k8s args - https://github.com/projectcalico/cni-plugin/blob/master/types/types.go#L89